src/components/display/fewlines/AlertBar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styled from "styled-components";
+
+import { IdentityTypes } from "../../../@types/Identity";
+
+export const AlertBar: React.FC<{ type: IdentityTypes }> = ({ type }) => {
+  if (type.toUpperCase() === IdentityTypes.EMAIL) {
+    return <Alert>Confirmation email has been sent</Alert>;
+  }
+  if (type.toUpperCase() === IdentityTypes.PHONE) {
+    return <Alert>Confirmation SMS has been sent</Alert>;
+  }
```

Comment:
```suggestion
  if (type === ReceivedIdentityTypes.EMAIL) {
    return <Alert>Confirmation email has been sent</Alert>;
  }
  if (type === ReceivedIdentityTypes.PHONE) {
    return <Alert>Confirmation SMS has been sent</Alert>;
  }
```

Reply:
OK fixed! fb3d867 only the `.toUpperCase()` were useless